docs/fiddles/features/recent-documents/main.js

```diff
@@ -0,0 +1,30 @@
+const { app, BrowserWindow } = require('electron')
+
+function createWindow () {
+  const win = new BrowserWindow({
+    width: 800,
+    height: 600,
+    webPreferences: {
+      nodeIntegration: true
+    }
+  })
+
+  win.loadFile('index.html')
+  win.webContents.openDevTools()
+}
+
+app.addRecentDocument('/Users/USERNAME/Desktop/work.type')
```

Comment:
Should we create a temporary file using `fs` and add it to the recent documents so that this example is runnable from the start?